tools/android_support/generate_pom_file.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
```

Comment:
Talked about this offline but so we don't forget - a comment somewhere at the top of this file explaining where it's used would be nice.

We should also update shell/platform/android/BUILD.gn to use this in the same PR

Reply:
Done. Thanks!